src/features/FEATURES-list.md

```diff
@@ -0,0 +1,48 @@
+
+# struktura
+- /src/features/NAMESPACE/FEATURE/*.files   - vsetky subory su takto povnarane na namespace (context) a featuru, vnutri su subory
+- /src/features/_shared/FEATURES/*          - v _shared su zdielane veci napriec projektom, a pod nim tiez features
+- /src/features/NAMESPACE/FEATURE/..        - vnutri featury sa moze nachadzat:
+    - SUBFEATURE/                           - ..nejaka subfeatura - je to lepsie delit takto, aby bolo jasne co to robi
+    - _types/                               - ..ak to neni nic konkretne, ale skor pomocne veci, tak podla typu s underscorom folder
+    - _components/                          - ..komponenty ktore nie su samostatnou featurou mozu byt takto
+    - _utils/                               - ..utilitky ktore nemaju jasnu featuru, sa daju odlozit sem
+
+
+# features/
```

Comment:
Toto odzrkadluje realne foldre? Cize ked to pribudne folder bude to treba pridat sem? Nie je to zbytocne trochu?

Reply:
odzrkadlovalo, suhlasim ze to moze byt zbytocny duplikat, pouzival som to hlavne pocas navrhovania tej struktury ze aby som mal prehlad, ked tak by sa dalo tam dat par slovami ze co ten folder je, a dat to ako snapshot z nejakeho datumu, taky log, tym padom nebude nevyhnutne to udrziavat, ale zaroven to moze priniest hodnotu ludom, ze sa trosku zorientuju

som ok aj s vymazanim, v aktualnom stave bez poznamok to nema pridanu hodnotu
takze zatial zmazem, a ak budes suhlasit so snapshot dokumentaciou, tak mozem to tam dorobit

Reply:
nechal som tam len namespaces, a vysvetlil co tam najdes - tie foldre pod tym uz su dost obvious v kontexte namespacu